Open-Rival/src/Rival.cpp

```diff
@@ -220,6 +225,13 @@ namespace Rival {
 
         //std::cout << mousePicker->getTileX() << ", " << mousePicker->getTileY() << "\n";
 
+        /// \todo Inspect the reason why sprites do not render correctly when this part is omitted
+        std::map<int, std::unique_ptr<Building>>& buildings = scenario->getBuildings();
+        for (auto const& kv : buildings) {
+            const std::unique_ptr<Building>& building = kv.second;
+            building->tick();
+        }
+
```

Comment:
What happens with the sprites when this is omitted?

Reply:
![2020-05-03_16-26-33](https://user-images.githubusercontent.com/36645163/80915442-101b8200-8d5b-11ea-85ba-768cdcdab2d4.png)

Reply:
🙀